### PR TITLE
fix: update doc and output console to generate https

### LINKED ISSUE #574: verdaccio ssl docs clarification
in the docs https://github.com/verdaccio/verdaccio/blob/master/docs/ssl.md
it says to generate the ssl files like so
```
 $ openssl genrsa -out ~/.config/verdaccio/verdaccio-key.pem 2048
 $ openssl req -new -sha256 -key ~/.config/verdaccio/verdaccio-key.pem -out ~/.config/verdaccio/verdaccio-csr.pem
 $ openssl x509 -req -in ~/.config/verdaccio/verdaccio-csr.pem -signkey ~/.config/verdaccio/verdaccio-key.pem -out ~/.config/verdaccio/verdaccio-cert.pem
```
and then reference them like so
```
https:
  key: ~/.config/verdaccio/server.key
  cert: ~/.config/verdaccio/server.crt
  ca: ~/.config/verdaccio/server.ca
```

But the 3 lines all out .pem files, not .key .crt .ca 

Am I missing that the files are renamed the verdaccio-key.pem to server.key etc?

### PR DESCRIPTION
**Pull request by juanpicado**
**Description:**

Improve ssl certs generation

Resolves #574

### CODE REVIEW

File: docs/ssl.md [hunk 1]
```diff
@@ -16,25 +16,25 @@ Once you update the listen and try to run verdaccio again will ask for certifica
 * Generate your certificates
 
 ````
- $ openssl genrsa -out ~/.config/verdaccio/verdaccio-key.pem 2048
- $ openssl req -new -sha256 -key ~/.config/verdaccio/verdaccio-key.pem -out ~/.config/verdaccio/verdaccio-csr.pem
- $ openssl x509 -req -in ~/.config/verdaccio/verdaccio-csr.pem -signkey ~/.config/verdaccio/verdaccio-key.pem -out ~/.config/verdaccio/verdaccio-cert.pem
+ $ openssl genrsa -out /Users/user/.config/verdaccio/verdaccio-key.pem 2048
```
Comment (jachstet-sea) [inline docs/ssl.md]: That looks a bit macOS-specific to me? Why did you remove the tilde which should be working fine on Linux and macOS?

Comment (juanpicado) [reply docs/ssl.md]: It was misleading, somehow only resolves absolute paths, I tried with the tilde and did not work. I put in in my todo, but in order to not confuse users, it has to be an absolute path.

Comment (jachstet-sea) [reply docs/ssl.md]: Sure, no problem :) I just wanted to make sure it was not changed to this by accident
